File: KuaforApi/Models/Review.cs
<<<<<<< HEAD
namespace KuaforApi.Models
{
    public class Review
    {
        public int Id { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; } = "";
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
=======
namespace KuaforApi.Models;

public class Review
{
    public int Id { get; set; }
    public int Rating { get; set; } // 1-5 arası
    public string? Comment { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    
    public int UserId { get; set; }
    public User? User { get; set; }
    
    public int SalonId { get; set; }
    public Salon? Salon { get; set; }
>>>>>>> bfce3bbeaa8220ef7a117b52532cece3251f9ef1
}

File: kuafor_app/lib/models/review.dart
class Review {
  final int? id;
  final int rating;
  final String comment;
  final DateTime createdAt;
  final int userId;
  final String? userName;
  final int salonId;

  Review({
    this.id,
    required this.rating,
    required this.comment,
    required this.createdAt,
    required this.userId,
    this.userName,
    required this.salonId,
  });

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
      id: json['id'],
      rating: json['rating'],
      comment: json['comment'],
      createdAt: DateTime.parse(json['createdAt']),
      userId: json['userId'],
      userName: json['user']?['fullName'],
      salonId: json['salonId'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'rating': rating,
      'comment': comment,
      'userId': userId,
      'salonId': salonId,
    };
  }
}
